tailcfg/tailcfg.go

```diff
@@ -526,11 +527,11 @@ type PortRange struct {
 
 var PortRangeAny = PortRange{0, 65535}
 
-// NetPortRange represents a single subnet:portrange.
+// NetPortRange represents a range of ports that's allowed for one or more IPs.
 type NetPortRange struct {
 	_     structs.Incomparable
-	IP    string // "*" means all
-	Bits  *int   // backward compatibility: if missing, means "all" bits
+	IP    string // IP, CIDR, Range, or "*" (same formats as FilterRule.SrcIPs)
```

Comment:
If I understand this correctly, we need to ensure that the server doesn't send a range if version < 7. Do we have any test in control for this? Any idea what it should look like?

Reply:
Yes, the mapCap stuff will be plumbed around. We don't need that until we start wanting to generate them.